_episodes/18-resources.md

```diff
@@ -113,7 +113,7 @@ get your job dispatched earlier.
 > > finish within 2 minutes:
 > >
 > > ```
-> > {{ site.sched.comment }} {{ site.sched.flag.time }}{% if site.sched.name == "Slurm" %} {% else %}={% endif %}00:02:00
+> > {{ site.sched.comment }} {{ site.sched.flag.time }}00:02:00
```

Comment:
```suggestion
> > {{ site.sched.comment }} {{ site.sched.flag.time }}{% if site.sched.name == "Slurm" or "SGE" %} {% else %}={% endif %}00:02:00
```
I believe this may work, but if we are going to diverge and then adopt what they have for a Slurm site in the future, then fine.